data/sql/updates/pending_db_world/rev_1623792334812044570.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1623792334812044570');
+
+SET @POOL = 60004;
+
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES (@POOL, 1, 'Water Barrel (1462) / Food Crate (10677) - Kharanos');
+
+INSERT INTO `pool_gameobject` (`guid`, `pool_entry`, `chance`, `description`) VALUES
```

Comment:
please use `DELETE` before `INSERT`

https://www.azerothcore.org/wiki/sql-standards

Reply:
Sorry, I missed that somehow. Will fix.